tests/unit/testHapiRequestInformationExtractor.js

```diff
@@ -123,34 +104,21 @@ test(
       , remoteAddress: ""
       , statusCode: 0
     };
-
-    t.plan(3);
-
-    t.deepEqual(
-      hapiRequestInformationExtractor(FULL_REQ_DERIVATION_VALUE)
-      , FULL_REQ_EXPECTED_VALUE
-      , [
-        "Given a full valid derivation value with an x-forwarded-for header the"
-        , "output of the extractor should reflect these values"
-      ].join(" ")
-    );
-
-    t.deepEqual(
-      hapiRequestInformationExtractor(PARTIAL_REQ_DERIVATION_VALUE)
-      , PARTIAL_REQ_EXPECTED_VALUE
-      , [
-        "Given a partial valid derivation value without an x-forwarded-for"
-        , "header the output of the extractor should reflect these values"
-      ].join(" ")
-    );
-
-    t.deepEqual(
-      hapiRequestInformationExtractor(ANOTHER_PARTIAL_REQ_DERIVATION_VALUE)
-      , ANOTHER_PARTIAL_REQ_EXPECTED_VALUE
-      , [
-        "Given a partial valid derivation value without a remote address the"
-        , "extractor should reflect these values"
-      ].join(" ")
-    );
-  }
-);
+    it('Should produce the expected value', function () {
```

Comment:
Under what reasoning? It still is a test which expects a specific value. Failure of the test would still point to the individual test